Packs/QutteraWebsiteMalwareScanner/Integrations/QutteraWebsiteMalwareScanner/QutteraWebsiteMalwareScanner.py

```diff
@@ -0,0 +1,208 @@
+    if not domain:
+        raise ValueError('domain is missing')
+
+    report = client._api_request(domain=domain, request_type="GET", operation="statscan")
+
+    return CommandResults(
+        outputs_prefix="QutteraWebsiteMalwareScanning.report",
+        outputs_key_field="error",
+        outputs=report
+    )
+
+
+def report_malware(client: Client, args: dict) -> CommandResults:
+    """
+    Returns malware status for given domain
+
+    """
+    domain = args.get("domain")
+    if not domain:
+        raise ValueError('domain is missing')
+
+    malware = client._api_request(domain=domain, request_type="GET", operation="malware")
+
+    return CommandResults(
+        outputs_prefix="QutteraWebsiteMalwareScanning.malware",
+        outputs_key_field="error",
+        outputs=malware
+    )
+
+
+def status_blacklist(client: Client, args: dict) -> CommandResults:
+    """
+    Returns url blacklisting information and reputation.
+
+    """
+    domain = args.get("domain")
+    if not domain:
+        raise ValueError('domain is missing')
+
+    start = client._api_request(domain=domain, request_type="GET", operation="blacklist")
+    readable_output = f"Blacklisted URI's of {domain}"
+    if start.get('errorstr') != 'success':
+        readable_output = f'Failed to get the report for domain {domain}'
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix="",
+        outputs_key_field='QutteraWebsiteMalwareScanning.domain',
+        outputs=start
+    )
+
+
+def test_module(client: Client):
+    """Performs basic request to ensure API server is running
+
+    Return True if request succeeded
+    """
+    resp = client._test_module()
+    if resp.get('errorstr') == 'success':
+        return 'ok'
+    raise DemistoException(f'Service is down: ${resp}')
+
+
+def main():  # pragma: no cover
+    """main function, parses params and runs command functions
+    """
+    params = demisto.params()
+    args = demisto.args()
+    command = demisto.command()
+    demisto.info(f"Command called {command}")
+    demisto.info(f"Args are {args}")
+    demisto.info(f"params are {params}")
+    client = Client(params)
+    if command == 'quttera-scan-start':
+        results = scan_start(client, args)
+    elif command == 'quttera-rescan-status':
+        results = rescan_status(client, args)
+    elif command == 'quttera-scan-report':
+        results = scan_report(client, args)
+    elif command == 'quttera-report-malware':
+        results = report_malware(client, args)
+    elif command == 'quttera-status-blacklist':
+        results = status_blacklist(client, args)
+    elif command == 'test-module':
+        results = test_module(client)
+    else:
+        raise NotImplementedError(f'Command {command} not implemented')
+    return_results(results)
+
+
+if __name__ in ('__main__', '__builtin__', 'builtins'):
+    try:
+        main()
+    except Exception as exception:
+        return_error(exception)
```

Comment:
```suggestion
def main():  # pragma: no cover
    """main function, parses params and runs command functions
    """
    params = demisto.params()
    args = demisto.args()
    command = demisto.command()
    demisto.info(f"Command called {command}")
    demisto.info(f"Args are {args}")
    demisto.info(f"params are {params}")
    client = Client(params)
    try:
      if command == 'quttera-scan-start':
          results = scan_start(client, args)
      elif command == 'quttera-rescan-status':
          results = rescan_status(client, args)
      elif command == 'quttera-scan-report':
          results = scan_report(client, args)
      elif command == 'quttera-report-malware':
          results = report_malware(client, args)
      elif command == 'quttera-status-blacklist':
          results = status_blacklist(client, args)
      elif command == 'test-module':
          results = test_module(client)
      else:
            raise ValueError(f'Command {command} is not implemented in this integration')
      return_results(results)
    except Exception as e:
        return_error(f'Failed to execute {command} command.\nError:\n{str(e)}')


if __name__ in ('__main__', '__builtin__', 'builtins'):
        main()
```

let's put the try and except inside the main only for the party where we ran the commands.